Source/CarthageKit/Xcode.swift

```diff
@@ -493,7 +494,7 @@ public func buildScheme( // swiftlint:disable:this function_body_length cyclomat
 			return !sdks.isEmpty
 		}
 		.flatMap(.concat) { platform, sdks -> SignalProducer<TaskEvent<URL>, CarthageError> in
-			let folderURL = workingDirectoryURL.appendingPathComponent(platform.relativePath, isDirectory: true).resolvingSymlinksInPath()
+			let folderURL = rootDirectoryURL.appendingPathComponent(platform.relativePath, isDirectory: true).resolvingSymlinksInPath()
```

Comment:
This also changed. Before, projects would copy their built binaries into their own `Carthage/Build/`, relying on the fact that it was symlinked to the root build dir. But if a project has no Carthage dependencies, this directory no longer exists. So this copies directly to the root dir.